doc/source/serve/doc_code/streaming_example.py

```diff
@@ -0,0 +1,38 @@
+# flake8: noqa
```

Comment:
How about using a real model as the example here?

Reply:
rather not do that here, it complicates communicating the feature. we can add a full example later